remote.py

```diff
@@ -53,12 +54,14 @@ def ReadFromFitbit(self, api_call, *args, **kwargs):
 		try:
 		 	resp = api_call(*args,**kwargs)
 		except HTTPTooManyRequests as e:
+			# retry between 5-10 minutes after the hour
```

Comment:
More like 5-10 minutes after the Fitbit quota reset. Right now this reset happens at the top of each hour but that's not a guarantee Fitbit offers us. No fix needed, just clarifying.